package.json

```diff
@@ -24,6 +24,7 @@
     "test_network_down": "docker-compose -f ./__tests__/integration/docker-compose.yml down",
     "dev": "nodemon --exec babel-node index.js",
     "start": "babel-node src/index.js",
+    "lint": "eslint . --fix",
```

Comment:
Shouldn't we run this in Github Actions?

Reply:
Good idea! Done on [318dad2](https://github.com/HathorNetwork/hathor-wallet-headless/pull/174/commits/318dad2802459a93004e60684240e0657e8dc702)